### PR TITLE
A little documentation

### PR DESCRIPTION
**Pull request by Yoric**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,62 @@
...
-`mx-tester` is a WIP tool to help test Matrix bots and Synapse modules.
\ No newline at end of file
+`mx-tester` is a WIP tool to help test Matrix bots and Synapse modules.
+
+# What `mx-tester` does
+
+The flow of `mx-tester` is the following:
+
+1. `mx-tester build` Create a Docker image for Synapse. If you need your Synapse to be setup with custom modules, you add scripts to this step to install the modules.
+2. `mx-tester up` Launch Synapse on port 8080.
```
Comment (Gnuxie) [inline README.md:10]: Well, the port is dependent on what they wrote in the config, but it probably doesn't matter

**Review by Gnuxie — APPROVED** [review]
Looks good, thanks :smiley:

File: README.md (hunk 1 above)
Comment (Yoric) [reply README.md:10]: Isn't it specified in the Dockerfile?

Comment (Yoric) [reply README.md]: Ah, my bad, it seems to be 9999.

**Pull request merged**